pkg/controller/manager/manager_controller.go

```diff
@@ -476,6 +481,20 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 	}
 
+	// Cloud modifications
+	mgrCfg := &corev1.ConfigMap{}
+	key := types.NamespacedName{Name: "cloud-manager-config", Namespace: rmeta.OperatorNamespace()}
+	if err = r.client.Get(ctx, key, mgrCfg); err != nil {
+		if !errors.IsNotFound(err) {
+			return reconcile.Result{}, fmt.Errorf("Failed to read cloud-manager-config ConfigMap: %s", err.Error())
+		}
+	} else {
+		if err = json.Unmarshal([]byte(mgrCfg.Data["config"]), &render.CloudManagerConfigOverrides); err != nil {
```

Comment:
I don't think there is any reason to involve a json.Unmarshall here. We should just store the portalAPIURL directly in the ConfigMap and then get it with
```
render.CloudManagerConfigOverrides = mgrCfg.Data["portalAPIURL"]
```
and then render.CloudManagerConfigOverrides is a string.
I think you copied this from the kibana config but that was json because for kibana we needed map. (now that I think about it, maybe I should have handled that differently for kibana and avoided json)

Reply:
Oh! Perfect, will do. thanks